pkg/agent/sidecar/service/restorer/restorer.go

```diff
@@ -212,6 +212,11 @@ func (r *restorer) restore(ctx context.Context) (err error) {
 
 		log.Debug("restoring: ", target)
 
+		if strings.Contains(target, "..") {
+			log.Warn(errors.ErrPathNotAllowed(target))
+			continue
```

Comment:
Just skipping the invalid file. is it okay?

Reply:
I think it's okay, because tr traverses directory we can skip upper directory

Reply:
thanks